homeassistant/components/ambee/models.py

```diff
@@ -1,15 +1,16 @@
 """Models helper class for the Ambee integration."""
 from __future__ import annotations
 
-from typing import TypedDict
+from typing import NamedTuple
 
 
-class AmbeeSensor(TypedDict, total=False):
+class AmbeeSensor(NamedTuple):
```

Comment:
What do you think about renaming it to `AmbeeSensorMetadata`?

Reply:
I'm against it, I don't think it is a good name.
As we have multiple places where e.g., lambda's are put in there as well, so I think metadata is wrong.

Besides, there is a difference in this integration between Sensors and SensorEntities.